src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -1050,4 +1030,32 @@ Status MediapipeGraphExecutor::inferStream(const ::inference::ModelInferRequest&
     }
 }
 
+Status MediapipeGraphExecutor::partialSerialize(const std::string& name, ::inference::ModelInferResponse& response, const ::mediapipe::Packet& packet) const {
```

Comment:
```suggestion
Status MediapipeGraphExecutor::serializePacket(const std::string& name, ::inference::ModelInferResponse& response, const ::mediapipe::Packet& packet) const {
```

@atobiszei @mzegla  during last sync you have stated that "partialSerialize" name should be changed, is it good?